cmake/ROCMExportTargetsHeaderOnly.cmake

```diff
@@ -95,11 +95,12 @@ function(rocm_export_targets_header_only)
 
     if(PARSE_TARGETS)
         rocm_write_package_template_function(${CONFIG_TEMPLATE} include "\${${PACKAGE_NAME}_TARGET_FILE}")
-# Disabled for PyTorch
-#        foreach(NAME ${PACKAGE_NAME} ${PACKAGE_NAME_UPPER} ${PACKAGE_NAME_LOWER})
-#            rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARIES ${PARSE_TARGETS})
-#            rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARY ${PARSE_TARGETS})
-#        endforeach()
+#        if(NOT (DEFINED PYTORCH_FOUND_HIP AND PYTORCH_FOUND_HIP) )
+#            foreach(NAME ${PACKAGE_NAME} ${PACKAGE_NAME_UPPER} ${PACKAGE_NAME_LOWER})
+#                rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARIES ${PARSE_TARGETS})
+#                rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARY ${PARSE_TARGETS})
```

Comment:
We can uncomment this, PYTORCH_FOUND_HIP should be available in pytorch and this can go in now.

Reply:
For this one I have a separate branch like you requested. The commit with the changes:
https://github.com/ROCmSoftwarePlatform/rocThrust/commit/4c87a56b986e4f2f89c1318d50c29d50c9ccea50

Reply:
In this PR we do not want to change the cmake/ROCMExportTargetsHeaderOnly.cmake file behaviour.